src/_scss/sharedComponents/_loadingMessage.scss

```diff
@@ -96,4 +97,41 @@
       }
   }
 
+  .results-table-error {
+    @include flex(1 1 auto);
+    margin-top: rem(100);
+    margin-bottom: rem(100);
+    max-width: rem(450);
+    padding-left: rem(10);
+    padding-right: rem(10);
+    margin-left: auto;
+    margin-right: auto;
+
+    .icon {
+        width: rem(75);
+        height: rem(72);
+        margin-left: auto;
+        margin-right: auto;
+
+        svg {
+            width: rem(72);
+            height: rem(72);
+            color: #d9534f;
```

Comment:
I'd recommend using one of our [color variables](https://github.com/fedspendingtransparency/data-act-broker-web-app/blob/dev/src/_scss/abstract/variables/colors.scss) for consistency with other elements. Looks like `$color-secondary-dark` is closest in value.